src/components/common/input/InputFile.tsx

```diff
@@ -0,0 +1,29 @@
+import File from '../file/File';
+import Uploader from '../uploader/Uploader';
+
+import { FileExtension } from '@/constants/file';
+
+interface InputFileProps {
+  fileList: File[];
+  deleteFile: (id: number) => void;
+  addFile: (file: FileList | null) => void;
```

Comment:
별도의 의도가 있지 않다면 콜백 함수를 가진 props의 경우 on~ 로 통일 시켜도 좋아보여요! 
```suggestion
  onDeleteFile: (id: number) => void;
  onAddFile: (file: FileList | null) => void;
```